app/models/region.rb

```diff
@@ -0,0 +1,33 @@
+class Region < ApplicationRecord
+  ltree :path
+  extend FriendlyId
+  friendly_id :name, use: :slugged
+
+  validates :name, presence: true
+  validates :slug, presence: true, uniqueness: true
```

Comment:
Don't think we can enforce uniqueness of the slug, can we? What happens if two blocks have the same name?

Reply:
Ugh.  True.

Reply:
Er, no, they need to be unique actually.  FriendlyID should be able to handle name dupes and append a number to get a unique one.  We need them unique so that we can use them for URLs and such.